apps/arweave/src/ar_metrics_collector.erl

```diff
@@ -56,7 +57,13 @@ metrics() ->
 	 {ar_header_sync_bytes_total, gauge, "ar_header_sync process memory",
 		get_process_memory(ar_header_sync)},
 	 {ar_wallets_bytes_total, gauge, "ar_wallets process memory",
-		get_process_memory(ar_wallets)}
+		get_process_memory(ar_wallets)},
```

Comment:
I know this long predates your PR (and even predates my time on the project), But I think we can remove all of the `message_queue_len` and process memory metrics. Unles @humaite you use them for something?

We currently track process memory and message queue length for all processes (not just the ones mentioned here) via the ar_process_sampler module (when `debug` is on).